dcos/config.py

```diff
@@ -213,6 +213,26 @@ def get_config_schema(command):
     return subcommand.config_schema(executable, command)
 
 
+def get_property_description(section, subkey):
+    """
+    :param section: section of config paramater
+    :type section: str
+    :param subkey: property within 'section'
+    :type subkey: str
+    :returns: description of section.subkey, or title if no description
+    :rtype: str
+    """
+
+    schema = get_config_schema(section)
+    property_info = schema["properties"].get(subkey)
+    if property_info is not None:
+        return property_info.get(
+            "description", property_info.get("title", "value"))
```

Comment:
Why don't we just return `None`? We can then check `None` in `dcos.http` so that we only printed if the value is found. What do you think?

```
description = config.get_property_description("core", "ssl_verify")
logger.exception("HTTP SSL Error")
msg = ("An SSL error occurred. To configure your SSL settings, "
            "please run: `dcos config set core.ssl_verify <value>`"
            "\n<value>: {}".format(description) if description else "")
raise DCOSException(msg)
```